compiler/front/cmdlinehelper.nim

```diff
@@ -125,18 +126,23 @@ proc writeConfigEvent(conf: ConfigRef,
       $sev
 
   let
-    severity =
+    diagSeverity =
       case evt.kind
       of cekInternalError:
-        styleSeverity configDiagSevFatal
+        configDiagSevFatal
       of cekLexerErrorDiag..cekFlagError:
-        styleSeverity configDiagSevError
+        configDiagSevError
       of cekLexerWarningDiag:
-        styleSeverity configDiagSevWarn
+        configDiagSevWarn
       of cekProgressConfStart, cekProgressPathAdded:
-        styleSeverity configDiagSevInfo # xxx: not really a hint
+        configDiagSevInfo # xxx: not really a hint
       of cekWriteConfig:
-        "" # not a log/diagnostic like the rest; has none
+        configDiagSevNone # not a log/diagnostic like the rest; has none
+
+    severity = styleSeverity diagSeverity
+
+  if diagSeverity <= configDiagSevError:
```

Comment:
(no change required) I regret the ordering of the enum, because it'd be so much nicer if this was a `>=`, oh well.